transactions.md

```diff
@@ -176,7 +176,12 @@ For greater detail, here's the process a retryable transaction goes through.
 
 	In some cases, the `RELEASE SAVEPOINT` statement itself can fail with a retryable error, mainly because transactions in CockroachDB only realize that they need to be restarted when they attempt to commit. If this happens, the retryable error is handled as described in step 4.
 
-## Transaction Priorities
+## Transaction Parameters
+
+Each transaction is controlled by two parameters: its priority and its
```

Comment:
What about them is controlled by those two parameters? Is this a functional grouping in some way?